### ISSUE TITLE
fix(notification): fix check mark & error icon color

### ISSUE THREAD

**Issue by jeanservaas** [carbon-design-system/carbon#3452]
We made the icon on the warning notification black for accessibility reasons but we need to also make the same changes for the error icon and the check mark icon.

Related to #3216

- [ ] need black fill for error icon for accessibility

- [ ] need black fill for check mark icon for accessibility

<img width="580" alt="image" src="https://user-images.githubusercontent.com/43144260/61477986-34d27480-a956-11e9-9113-1017d3febbb1.png">

**Comment by asudoh** [comment]
@jeanservaas Should we merge this issue to #3443? Thanks!

**Comment by jeanservaas** [comment]
@asudoh hahah yes, same issue sorry, how do we merge? shall i just close this?

**Comment by jeanservaas** [comment]
Closed in favor of #3443

**Comment by asudoh** [comment]
Thanks @jeanservaas!

**Comment by mjabbink** [comment]
Also recommend bringing back the pastel ones with outline as an option. Those are also very nice and work better in certain situations

**Comment by mjabbink** [comment]
@jeanservaas I’m pretty sure the white icons Work in 50s no? The new v2 was all about getting those to work but maybe I’m wrong.
